src/java.desktop/windows/native/libawt/windows/awt_ScrollPane.cpp

```diff
@@ -525,6 +525,7 @@ void AwtScrollPane::_SetSpans(void *param)
             parentWidth, parentHeight, childWidth, childHeight);
         s->RecalcSizes(parentWidth, parentHeight, childWidth, childHeight);
         s->VerifyState();
+        s->SetInsets(env);
```

Comment:
I guess we should call "s->VerifyState();" after the "s->SetInsets(env);" similar to what we do in the "AwtScrollPane::_SetInsets"

Reply:
> I guess we should call "s->VerifyState();" after the "s->SetInsets(env);" similar to what we do in the "AwtScrollPane::_SetInsets"

@mrserb An interesting question… You're right, I've introduced an inconsistency.

Yet I don't think it matters because `SetInsets` doesn't change any members of `AwtScrollPane` object which are asserted in `VerifyState`. In addition to it, `VerifyState` is a no-op in release builds.

I'll take this into account in [JDK-8310056](https://bugs.openjdk.org/browse/JDK-8310056).